tests/dummy/app/controllers/index.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Controller.extend({
 
   allCookies: computed(function() {
     let cookieService = this.get('cookies');
-    cookieService.write('now', new Date().getTime());
+    cookieService.write('now', new Date().getTime(), { domain: 'localhost' });
```

Comment:
I will try to get a cycle tomorrow and create some tests for this and remove the domain from the dummy.